src/modules/UI/components/Header/Component/WalletNameHeader.ui.js

```diff
@@ -8,24 +8,19 @@ type Props = {
   styles: {
     textStyles: Array<{}>
   },
-  name: string,
-  denomination: string,
-  walletDirectionString?: string
+  name: string
 }
 
 class WalletNameHeader extends React.Component<Props> {
   render () {
     const { styles = {} } = this.props
     const textStyles = styles.textStyles || []
     const name = this.props.name
-    const denomination = this.props.denomination
-    const directionPrefix = this.props.walletDirectionString ? this.props.walletDirectionString : ''
 
     return (
       <View style={style.headerNameContainer}>
         <Text style={textStyles} ellipsizeMode={'middle'} numberOfLines={1}>
-          {directionPrefix + ' ' + name}
-          <Text style={[style.cCode, ...textStyles]}> ({denomination})</Text>
+          {name}
```

Comment:
You removed the denomination from the header. I don't believe this should be removed